lib/triton-tags.js

```diff
@@ -0,0 +1,185 @@
+        assert.string(val, 'val');
+        if (val.length > 255 || !val.match(DNS_NAME_RE)) {
+            throw new Error(format(
+                'invalid "triton.cns.reverse_ptr" tag: "%s" is not DNS safe',
+                val));
+        }
+    }
+};
+
+
+// --- exports
+
+/*
+ * Return true if the given key uses the Triton tag prefix.
+ * Note that it still might not be one of the specific defined tag
+ *
+ */
+function isTritonTag(key) {
+    assert.string(key, 'key');
+    return (key.substr(0, TRITON_TAG_PREFIX.length) === TRITON_TAG_PREFIX);
+}
+
+
+/**
+ * Validate the given Triton tag key and value. This differs from
+ * `validateTritonTagStr` in that the `val` argument should already be of
+ * the correct type.
+ *
+ * @returns {String} On success, the validated value is returned.
+ * @throws {Error} if the key is an unknown tag or val is invalid.
+ */
+function validateTritonTag(key, val) {
+    assert.string(key, 'key');
+
+    // Check type.
+    var expectedType = typeFromKey[key];
+    var actualType = typeof (val);
+    if (expectedType === undefined) {
+        throw new Error('Unrecognized special triton tag "' + key + '"');
+    } else if (expectedType !== actualType) {
+        throw new Error(format('Triton tag "%s" must be a %s: %j (%s)',
+            key, expectedType, val, actualType));
+    }
+
+    var validator = validatorFromKey[key];
+    assert.func(validator, 'validator for tag ' + key);
+    validator(val);
+
+    return val;
+}
+
+/*
+ * Convert and validate the given Triton tag string value.
+ *
+ * @returns {String} On success, the validated value is returned.
+ * @throws {Error} if the key is an unknown tag or val is of the wrong type
+ *      or is invalid.
+ */
+function validateTritonTagStr(key, str) {
+    assert.string(key, 'key');
+    assert.string(str, 'str');
+
+    // Ensure it is a known triton tag.
+    var type = typeFromKey[key];
+    if (type === undefined) {
+        throw new Error('Unrecognized special triton tag "' + key + '"');
+    }
+
+    // Convert from string to value of appropriate type.
+    var val;
+    switch (type) {
+    case 'string':
+        val = str;
+        break;
+    case 'boolean':
+        if (str === 'true') {
+            val = true;
+        } else if (str === 'false') {
+            val = false;
+        } else {
+            throw new Error(format(
+                'Triton tag "%s" value must be "true" or "false": %j',
+                key, str));
+        }
+        break;
+    case 'number':
+        val = Number(str);
+        if (isNaN) {
+            throw new Error(format('Triton tag "%s" value must be a number: %j',
+                key, str));
+        }
+        break;
+    default:
+        throw new Error('unexpected Triton tag type: ' + type);
+    }
+
+    // Validate.
+    var validator = validatorFromKey[key];
+    assert.func(validator, 'validator for tag ' + key);
+    validator(val);
+
+    return val;
+}
+
+
+module.exports = {
```

Comment:
The only reason would be if you want to be able to change it. But as I said above, that's the only good reason I see for having a constant at all. Since we're not going to make it possible to change, there's no good reason to export it. Everyone can just hardcode 'triton.' and be happy.